<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MarkDown Editor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources/markdowneditor.qrc">
    <normaloff>:/3rdparty/breeze-icon/default.png</normaloff>:/3rdparty/breeze-icon/default.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QPlainTextEdit" name="editor">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="backgroundVisible">
        <bool>false</bool>
       </property>
       <property name="centerOnScroll">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QTextBrowser" name="textBrowser">
       <property name="styleSheet">
        <string notr="true">body{
  margin: 0 auto;
  font-family: serif,Georgia;
  color: #444444;
  line-height: 1;
  max-width: 90%;
  padding: 30px;
}
h1, h2, h3, h4 {
  color: #111111;
  font-weight: 400;
}
h1, h2, h3, h4, h5, p {
  margin-bottom: 24px;
  padding: 0;
}
h1 {
  font-size: 48px;
}
h2 {
  font-size: 36px;
  /* The bottom margin is small. It's designed to be used with gray meta text
   * below a post title. */
  margin: 24px 0 6px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 21px;
}
h5 {
  font-size: 18px;
}
a {
  color: #0099ff;
  margin: 0;
  padding: 0;
  vertical-align: baseline;
}
a:hover {
  text-decoration: none;
  color: #ff6600;
}
a:visited {
  color: purple;
}
ul, ol {
  padding: 0;
  margin: 0;
}
li {
  line-height: 24px;
}
li ul, li ul {
  margin-left: 24px;
}
p, ul, ol {
  font-size: 16px;
  line-height: 24px;
  max-width: 90%;
}
pre {
  padding: 0px 24px;
  max-width: 90%;
  white-space: pre-wrap;
}
code {
  font-family: Consolas, Monaco, Andale Mono, monospace;
  line-height: 1.5;
  font-size: 13px;
  background-color: #f8f8f8;
}
pre{
  background-color: #f8f8f8;
  border: 1px solid #ccc;
  padding: 6px 10px;
  border-radius: 3px;
}
aside {
  display: block;
  float: right;
  width: 390px;
}
blockquote q{
  border-left:.5em solid #eee;
  padding: 0 2em;
  margin-left:0;
  max-width:85%;
}
blockquote  cite {
  font-size:14px;
  line-height:20px;
  color:#bfbfbf;
}
blockquote cite:before {
  content: '\2014 \00A0';
}

blockquote p {
  color: #666;
  max-width: 80%;
}
hr {
  text-align: left;
  margin: 0 auto 0 0;
  color: #999;
}

/* Code below this line is copyright Twitter Inc. */

/*button,
input,
select,
textarea {
font-size: 100%;
margin: 0;
vertical-align: baseline;
*vertical-align: middle;
}
button, input {
line-height: normal;
*overflow: visible;
}
button::-moz-focus-inner, input::-moz-focus-inner {
border: 0;
padding: 0;
}
input[type=checkbox], input[type=radio] {
cursor: pointer;
}
/* override default chrome &amp; firefox settings
input:not([type=&quot;image&quot;]), textarea {
box-sizing: content-box;
}

input[type=&quot;search&quot;] {
box-sizing: content-box;
}
label,
select,
textarea {
font-family: &quot;Helvetica Neue&quot;, Helvetica, Arial, sans-serif;
font-size: 13px;
font-weight: normal;
line-height: normal;
margin-bottom: 18px;
}
input[type=checkbox], input[type=radio] {
cursor: pointer;
margin-bottom: 0;
}
textarea {
height: auto;
}

input[type=text],
input[type=password],
select,
textarea {
transition: border linear 0.2s, box-shadow linear 0.2s;
box-shadow: inset 0 1px 3px rgba(0, 0, 0, 0.1);
}
input[type=text]:focus, input[type=password]:focus, textarea:focus {
outline: none;
transform: translate(-2px,-2px);
background-color: #fff;
transition: background-color .1s,box-shadow .1s,color .1s,transform .1s;
box-shadow: 4px 4px 0 0 #000;
color: #000;
}*/

/* black */
button,
input[type=&quot;button&quot;],
input[type=&quot;reset&quot;],
input[type=&quot;submit&quot;]{
      -webkit-box-sizing: border-box;
      -moz-box-sizing: border-box;
      box-sizing: border-box;
      font-size: 16px;
      font-size: 1rem;
      line-height: 1.5;
      transition: background-color .1s,box-shadow .1s,color .1s,transform .1s;
      background-color: #000;
      border-radius: 2px;
      border: 2px solid #000;
      color: #fff;
      cursor: pointer;
      display: inline-block;
      font-weight: 700;
      padding: 14px 30px;
      text-decoration: none !important;
}
button:hover {
      transform: translate(-2px,-2px);
      background-color: #fff;
      box-shadow: 4px 4px 0 0 #000;
      color: #000;
}

/*white*/
textarea,input:not([type=&quot;radio&quot;]):not([type=&quot;range&quot;]):not([type=&quot;checkbox&quot;]),select{
-webkit-box-sizing: border-box;
-moz-box-sizing: border-box;
box-sizing: border-box;
font-size: 16px;
font-size: 1rem;
line-height: 1.5;
transition: background-color .1s,box-shadow .1s,color .1s,transform .1s;
background-color: #fff;
border-radius: 2px;
border: 2px solid #000;
color: #000;
cursor: pointer;
display: inline-block;
font-weight: 700;
padding: 14px 30px;
text-decoration: none !important;
}
textarea:hover,input:not([type=&quot;radio&quot;]):not([type=&quot;range&quot;]):not([type=&quot;checkbox&quot;]):hover,select:hover {
transform: translate(-2px,-2px);
background-color: #fff;
box-shadow: 4px 4px 0 0 #000;
color: #000;
}


input[type=&quot;range&quot;] {
/*-webkit-box-shadow: 0 1px 0 0px #424242, 0 1px 0 #060607 inset, 0px 2px 10px 0px black inset, 1px 0px 2px rgba(0, 0, 0, 0.4) inset, 0 0px 1px rgba(0, 0, 0, 0.6) inset;*/
margin-top: 42px;
background-color: #ebeff4;
-moz-appearance: none;
-webkit-appearance: none;
height:4px;
padding: 0;
border: none;
}
</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuSet_Codec">
     <property name="title">
      <string>Open with Codec</string>
     </property>
     <addaction name="actionCodecAuto"/>
     <addaction name="actionReadUTF_8"/>
     <addaction name="actionReadGBK"/>
     <addaction name="actionReadOthers"/>
    </widget>
    <widget class="QMenu" name="menuSave_with_Codec">
     <property name="title">
      <string>Save with Codec</string>
     </property>
     <addaction name="actionSaveUTF_8"/>
     <addaction name="actionSaveGBK"/>
     <addaction name="actionSaveOther"/>
    </widget>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="menuSet_Codec"/>
    <addaction name="actionSave"/>
    <addaction name="menuSave_with_Codec"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionFont"/>
    <addaction name="actionFind"/>
    <addaction name="actionWC"/>
    <addaction name="separator"/>
    <addaction name="actionPreview"/>
    <addaction name="actionView_mode"/>
    <addaction name="separator"/>
    <addaction name="actionShowHTML"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionHow"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_2"/>
   <addaction name="actionOpen_2"/>
   <addaction name="actionSave_2"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resources/markdowneditor.qrc">
     <normaloff>:/3rdparty/breeze-icon/document-open.png</normaloff>:/3rdparty/breeze-icon/document-open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="toolTip">
    <string>Open document</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resources/markdowneditor.qrc">
     <normaloff>:/3rdparty/breeze-icon/document-save.png</normaloff>:/3rdparty/breeze-icon/document-save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="toolTip">
    <string>Save current document</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="resources/markdowneditor.qrc">
     <normaloff>:/3rdparty/breeze-icon/application-exit.png</normaloff>:/3rdparty/breeze-icon/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="toolTip">
    <string>Exit editor</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="resources/markdowneditor.qrc">
     <normaloff>:/3rdparty/breeze-icon/document-save-as.png</normaloff>:/3rdparty/breeze-icon/document-save-as.png</iconset>
   </property>
   <property name="text">
    <string>Save &amp;As...</string>
   </property>
   <property name="toolTip">
    <string>Save document under different name</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="resources/markdowneditor.qrc">
     <normaloff>:/3rdparty/breeze-icon/document-new.png</normaloff>:/3rdparty/breeze-icon/document-new.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="toolTip">
    <string>Create new document</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="icon">
    <iconset resource="resources/markdowneditor.qrc">
     <normaloff>:/3rdparty/breeze-icon/gtk-select-font.png</normaloff>:/3rdparty/breeze-icon/gtk-select-font.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Fonts...</string>
   </property>
   <property name="toolTip">
    <string>Change Fonts</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="icon">
    <iconset resource="resources/markdowneditor.qrc">
     <normaloff>:/3rdparty/breeze-icon/edit-find.png</normaloff>:/3rdparty/breeze-icon/edit-find.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Find</string>
   </property>
   <property name="toolTip">
    <string>Find something</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionHow">
   <property name="text">
    <string>&amp;How to</string>
   </property>
   <property name="toolTip">
    <string>How to use markdown</string>
   </property>
  </action>
  <action name="actionWC">
   <property name="text">
    <string>Word &amp;Count</string>
   </property>
   <property name="toolTip">
    <string>count how much word </string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="resources/markdowneditor.qrc">
     <normaloff>:/3rdparty/breeze-icon/help-about.png</normaloff>:/3rdparty/breeze-icon/help-about.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionPreview">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Preview</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionNew_2">
   <property name="icon">
    <iconset resource="resources/markdowneditor.qrc">
     <normaloff>:/3rdparty/breeze-icon/document-new.png</normaloff>:/3rdparty/breeze-icon/document-new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen_2">
   <property name="icon">
    <iconset resource="resources/markdowneditor.qrc">
     <normaloff>:/3rdparty/breeze-icon/document-open.png</normaloff>:/3rdparty/breeze-icon/document-open.png</iconset>
   </property>
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionSave_2">
   <property name="icon">
    <iconset resource="resources/markdowneditor.qrc">
     <normaloff>:/3rdparty/breeze-icon/document-save.png</normaloff>:/3rdparty/breeze-icon/document-save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionView_mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>View mode</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionCodecAuto">
   <property name="text">
    <string>&amp;Auto Detecet Codec</string>
   </property>
  </action>
  <action name="actionReadUTF_8">
   <property name="text">
    <string>UTF-8</string>
   </property>
  </action>
  <action name="actionReadGBK">
   <property name="text">
    <string>GBK(GB 18310)</string>
   </property>
  </action>
  <action name="actionReadOthers">
   <property name="text">
    <string>&amp;Others...</string>
   </property>
   <property name="toolTip">
    <string>Type a Codec </string>
   </property>
  </action>
  <action name="actionSaveUTF_8">
   <property name="text">
    <string>UTF-8</string>
   </property>
  </action>
  <action name="actionSaveGBK">
   <property name="text">
    <string>GBK(GB 18310)</string>
   </property>
   <property name="toolTip">
    <string>GBK(GB 18310)</string>
   </property>
  </action>
  <action name="actionSaveOther">
   <property name="text">
    <string>&amp;Other...</string>
   </property>
  </action>
  <action name="actionShowHTML">
   <property name="icon">
    <iconset resource="resources/markdowneditor.qrc">
     <normaloff>:/3rdparty/breeze-icon/dialog-xml-editor.png</normaloff>:/3rdparty/breeze-icon/dialog-xml-editor.png</iconset>
   </property>
   <property name="text">
    <string>Show as HTML</string>
   </property>
   <property name="toolTip">
    <string>Show as HTML</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources/markdowneditor.qrc"/>
 </resources>
 <connections/>
</ui>
